### DISCUSSION TITLE
how to use overlapping

### DISCUSSION THREAD

**Discussion by wqingzex** [mapequation/infomap#228] [Q&A] [answered]
using infomap 1.3.0，python code
```
infomap.Infomap("--overlapping")
```
 then output
```
Infomap version 1.3.0
Unrecognized option: '--overlapping'
Usage: Infomap out_directory [options]. Run with option '-h' for more information.
```

so how to use overlapping?

my dataset contain thousands of videos，I want to give more than one tag to each video, in fact it does, by clustering

**Comment by antoneri** [comment, marked as answer]
Hi, and thanks for reaching out!

I converted this issue to a discussion as we try to separate bugs from questions.

To your question:

In old Infomap, we supported overlapping assignments by "faking" memory in ordinary first-order networks. 

This made a lot of assumptions, and we decided to not include it in the "new" Infomap (1.0 and upwards).

Model your data as multilayer, memory, or raw state networks if you want overlapping assignments. See https://www.mapequation.org/infomap/ section "Input formats".

Hope this helps!

**Reply by wqingzex** [reply]
OK，do you mean that if I model my dataas multilayer, then the result is overlapping, nothing to do any more

**Reply by antoneri** [reply]
Yes, that is correct!

You'd have to think about what constitutes a layer in your case, though.

**Reply by wqingzex** [reply]
yes, that's the big problem,
thanks
